bpython/curtsiesfrontend/interpreter.py

```diff
@@ -60,7 +60,9 @@ def format(self, tokensource, outfile):
 
 
 class Interp(ReplInterpreter):
-    def __init__(self, locals: Dict[str, Any] = None, encoding=None):
+    def __init__(
+        self, locals: Dict[str, Any] = None, encoding: str = None
```

Comment:
This should actually be marked with `Optional`.

Reply:
… ah, but this is just noise from reformatting. This change will need to be rebased.